packages/scalecube-microservice/tests/implementations/Microservice.spec.ts

```diff
@@ -100,14 +145,40 @@ describe('Test the creation of Microservice', () => {
         services: [greetingService1, greetingService2],
       });
 
-      test('Test REQUEST_RESPONSE', () => {
+      test(`
+      Scenario: Invoke registered serviceS from proxy (RequestResponse)
+        Given   a service with definition and reference
+        And     definition and reference comply with each other
+        |service          |definition            |reference             |
+        |greetingService1 |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |
+        |greetingService2 |hello: RequestResponse|hello: RequestResponse|
+        |                 |greet$: RequestStream |greet$: RequestStream |      
```

Comment:
Change to:
|service          |definition            |reference             |
|greetingService1 |greet$: RequestStream |greet$: RequestStream |
|greetingService2 |hello: RequestResponse|hello: RequestResponse|